src/megaapi_impl.cpp

```diff
@@ -10776,9 +10776,16 @@ bool MegaApiImpl::processMegaTree(MegaNode* n, MegaTreeProcessor* processor, boo
     return result;
 }
 
-MegaNodeList* MegaApiImpl::searchInAllShares(const char *searchString, MegaCancelToken *cancelToken, int order, int target)
+MegaNodeList* MegaApiImpl::searchInAllShares(const char *searchString, MegaCancelToken *cancelToken, int order, int target, int type)
```

Comment:
I'm not sure but this method can't be replaced by `MegaNodeList* MegaApiImpl::search(MegaNode *n, const char* searchString, MegaCancelToken *cancelToken, bool recursive, int order, int type, int target) ` with this parameters: `search(nullptr, searchString, cancelToken, true, order, TARGET_INSHARE | TARGET_OUTSHARE, type)